tsconfig.json

```diff
@@ -32,6 +32,6 @@
     "@datalib/*": ["./app/(api)/_datalib/*"],
     }
   },
-  "include": ["nextEnv.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
+  "include": ["nextEnv.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts", "app/(api)/api/testCollection/post.js", "app/(api)/api/testCollection/get.js", "app/(api)/api/testCollection/route.js"],
```

Comment:
i think we can get rid of this change. for some reason my typescript config adds this automatically but i guess just remove the change from the PR. I'll see if I can get a permanent fix going later.